app/src/services/object.js

```diff
@@ -85,6 +85,7 @@ const service = {
    * @param {string} [params.originalName] Optional filename string to match on
    * @param {string} [params.path] Optional canonical S3 path string to match on
    * @param {string} [params.mimeType] Optional mimeType string to match on
+   * @param {string} [params.metadata] Optional metadata string to match on
```

Comment:
If we're not implementing the search side of things just yet, let's not add this in just yet to the JSDocs.